runtime/interface.cpp

```diff
@@ -2293,6 +2301,8 @@ static void reset_global_interface_vars(PhpScriptBuiltInSuperGlobals &supergloba
 }
 
 static void init_interface_lib() {
+  headers_sent = false;
```

Comment:
Why var `headers_sent` set false here?

Reply:
@astrophysik 
As I see it's an appropriate practice in case when some state releasing scenarios (e.g. `free_interface_lib()`) potentially may be not finished properly. 
You may see the same manipulation with `shutdown_functions_count` -- it is also double reinitializing in `init_interface_lib`. 

I can remove it if you want

Reply:
@astrophysik Fixed